daemon/logger/journald/journald.go

```diff
@@ -1,5 +1,7 @@
 // +build linux
 
+// Package journald provides the log driver for forwarding server logs
+// to endpoints that recieve the systemd format.
```

Comment:
well, this is embarrassing, so I've now figured out how to enable spell checking. Thanks